frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -39,19 +39,23 @@ export default class DocumentListToolbar extends Component {
     onSave: proptypes.func,
 
     /**
-     * The document currently being edited.
+     * An array of peers currently editing the document.
      */
     peers: proptypes.object
   }
 
+  static defaultProps = {
+    disabled: false
+  }
+
   constructor(props) {
     super(props)
   }
 
   render() {
     const {
+      disabled,
```

Comment:
@eduardoboucas why are we consolidating both validationErrors and connectionErrors here? Why would we stop deleting if the document had validation errors?